tfx/components/example_gen/utils.py

```diff
@@ -214,9 +223,39 @@ def _glob_to_regex(glob_pattern: Text) -> Text:
   return regex_pattern
 
 
+def _verify_split_pattern_specs(
+    split: example_gen_pb2.Input.Split) -> Tuple[bool, bool, bool]:
+  """Verify and identify specs to be matched in split pattern."""
+  is_match_span = SPAN_SPEC in split.pattern
+  is_match_date = any(spec in split.pattern for spec in DATE_SPECS)
+  is_match_version = VERSION_SPEC in split.pattern
+
+  if [is_match_span, is_match_date].count(True) > 1:
+    raise ValueError('Either span spec or date specs must be specified '
+                     'exclusively.')
```

Comment:
add the split.pattern to the error string like other cases, which helps users to fix configs

Reply:
fixed